torch/serialization.py

```diff
@@ -656,6 +656,9 @@ def persistent_id(obj):
     data_value = data_buf.getvalue()
     zip_file.write_record('data.pkl', data_value, len(data_value))
 
+    # Write byte order marker
+    zip_file.write_record('byteorder', sys.byteorder, len(sys.byteorder))
```

Comment:
I wonder if `torch.save` should have a flag `_disable_byteorder_record` just so we can get the old behavior again? I guess probably not though. If someone ends up asking for that, we can always just add that in later on

Reply:
Oh right, that's a good point